lambdas/service/app.py

```diff
@@ -86,7 +86,77 @@
 
 openapi_spec = {
     'info': {
-        'description': 'This should probably be really long',
+        'description': format_description('''
+            # Overview
+            Azul provides a programmatic interface for quickly and uniformly
+            querying metadata associated with experimental and analysis data
+            stored in the
+            [data store (DSS)](https://github.com/HumanCellAtlas/data-store).
+            Azul serves as the backend for the
+            [data-browser](https://github.com/DataBiosphere/data-browser).
+
+            Azul aggregates metadata into several different indices:
+             - [files](#operations-Repository-get_repository_files)
+             - [samples](#operations-Repository-get_repository_samples)
+             - [bundles](#operations-Repository-get_repository_bundles)
+
+            which are all stored in respective Elasticsearch indices.
+
+            Azul provides the ability to download relevant data via the
+            [Manifests](#/Manifests) endpoints. After refining a search with the
+            [Repository](#/Repository) endpoints, these can be used to download
+            a manifest of the relevant files. The manifest can then be used by
+            the [DCP CLI](https://github.com/HumanCellAtlas/dcp-cli) to download
+            the listed files.
+
+            Azul also provides a
+            [summary](#operations-Repository-get_repository_summary) view of
+            indexed data.
+
+            ## Data model
+            Each index, when queried returns a JSON array of _entities_. An
+            entity is comprised of a handful of different kinds of _inner
+            entities_. Inner entities contains concrete metadata fields. Each
+            inner entity corresponds to a metadata object in the original
+            metadata graph.
```

Comment:
We need to drive home the point that inner entities are actually connected to or associated with the outer entity via the graph. Only properties from graph-connected entities are represented as properties of inner entities in each hit.